stock_valuation_layer_usage/models/stock_move.py

```diff
@@ -46,3 +46,12 @@ def _create_dropshipped_svl(self, forced_quantity=None):
                 }
             )
         return layers
+
+    def _get_in_moves(self):
```

Comment:
I'd prefer returning ids as better in terms of performances to do a set of ids and not using `|` operator